<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkeletonBasedAnimationUI</class>
 <widget class="QFrame" name="SkeletonBasedAnimationUI">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <property name="tabsClosable">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="m_player">
      <attribute name="title">
       <string>Player</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="m_speed_l">
            <property name="text">
             <string>Speed:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="m_speed">
            <property name="minimum">
             <double>0.010000000000000</double>
            </property>
            <property name="maximum">
             <double>100.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="m_autoRepeat">
            <property name="text">
             <string>Auto repeat</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QCheckBox" name="m_pingPong">
            <property name="text">
             <string>Ping pong</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Editor">
      <attribute name="title">
       <string>Edition</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox_2_1">
         <property name="title">
          <string>Animation</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2_1">
          <item row="0" column="0">
           <widget class="QLabel" name="m_animationID_l">
            <property name="text">
             <string>Current:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="m_newAnim">
            <property name="text">
             <string>New</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="m_removeAnim">
            <property name="text">
             <string>Remove</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QComboBox" name="m_currentAnimation"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>File</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Current:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="m_animFile">
              <property name="text">
               <string>none</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="m_saveAnim">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="m_loadAnim">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_rig">
      <attribute name="title">
       <string>Skeleton</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>X-Ray:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Show:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="m_xray">
         <property name="minimumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/Resources/Icons/xray.png</normaloff>
           <normalon>:/Resources/Icons/xray.png</normalon>:/Resources/Icons/xray.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="m_showSkeleton">
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <spacer name="verticalSpacer_1">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="m_manipulation">
         <item>
          <property name="text">
           <string>Forward</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Pseudo IK</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Manipulation Scheme</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="m_skinning">
      <attribute name="title">
       <string>Skinning</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <widget class="QComboBox" name="m_weightsType">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Standard Weights</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>STBS Weights</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Normal Skinning:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="m_skinningMethod">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <item>
          <property name="text">
           <string>Linear Blend Skinning</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dual Quaternion Skinning</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Center of Rotation Skinning</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>STBS with LBS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>STBS with DQS</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Skinning Method:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="m_normalSkinning">
         <item>
          <property name="text">
           <string>Standard Approx.</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Geometric Eval.</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="m_showWeights">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Show Weights</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionXray">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset theme="xray.png" resource="SkeletonBasedAnimation.qrc">
     <normaloff>:/Icons/xray.png</normaloff>:/Icons/xray.png</iconset>
   </property>
   <property name="text">
    <string>xray</string>
   </property>
   <property name="toolTip">
    <string>Show / hide skeleton</string>
   </property>
   <property name="shortcut">
    <string>S</string>
   </property>
  </action>
  <action name="actionLBS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="SkeletonBasedAnimation.qrc">
     <normaloff>:/Icons/LB.png</normaloff>
     <normalon>:/Icons/LB_on.png</normalon>:/Icons/LB.png</iconset>
   </property>
   <property name="text">
    <string>LBS</string>
   </property>
   <property name="toolTip">
    <string>LBS</string>
   </property>
  </action>
  <action name="actionDQS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="SkeletonBasedAnimation.qrc">
     <normaloff>:/Icons/DQ.png</normaloff>
     <normalon>:/Icons/DQ_on.png</normalon>:/Icons/DQ.png</iconset>
   </property>
   <property name="text">
    <string>DQS</string>
   </property>
  </action>
  <action name="actionCoR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="SkeletonBasedAnimation.qrc">
     <normaloff>:/Icons/CoR.png</normaloff>
     <normalon>:/Icons/CoR_on.png</normalon>:/Icons/CoR.png</iconset>
   </property>
   <property name="text">
    <string>CoR</string>
   </property>
  </action>
  <action name="actionSTBSLBS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="SkeletonBasedAnimation.qrc">
     <normaloff>:/Icons/STBSLB.png</normaloff>
     <normalon>:/Icons/STBSLB_on.png</normalon>:/Icons/STBSLB.png</iconset>
   </property>
   <property name="text">
    <string>STBSLBS</string>
   </property>
  </action>
  <action name="actionSTBSDQS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="SkeletonBasedAnimation.qrc">
     <normaloff>:/Icons/STBSDQ.png</normaloff>
     <normalon>:/Icons/STBSDQ_on.png</normalon>:/Icons/STBSDQ.png</iconset>
   </property>
   <property name="text">
    <string>STBSDQS</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="SkeletonBasedAnimation.qrc"/>
 </resources>
 <connections/>
</ui>
